### PR TITLE
Added patch for linux 6.11 kernel support

### PR DESCRIPTION
**Pull request by mortanius-1**
Thanks for this; here's a patch for Kernel 6.11 (6.11.4-201.fc40.x86_64).

### CODE REVIEW

File: prepare-for-current-kernel.sh [hunk 1]
```diff
@@ -7,14 +7,14 @@ then
 elif { echo $VERSION ; echo "5.99" ; } | sort -V -c 2>/dev/null
 then
   PATCHFILE="patch5.8"
-elif { echo $VERSION ; echo "6.19" ; } | sort -V -c 2>/dev/null
+elif { echo $VERSION ; echo "6.10" ; } | sort -V -c 2>/dev/null
 then
   PATCHFILE="patch6.0"
 else
-  PATCHFILE="patch6.2"
+  PATCHFILE="patch6.11"
```
Comment (Plippo) [inline prepare-for-current-kernel.sh:14]: Is there a reason you removed the branch for 6.2?

**Review by Plippo — CHANGES_REQUESTED** [review]

File: prepare-for-current-kernel.sh (hunk 1 above)
Comment (mortanius-1) [reply prepare-for-current-kernel.sh:14]: Hi Plippo no reason it was a quick way to test and validate the patch when I tested it on my Zenbook. I actually intended to only submit the patch file and not `prepare-for-current-kernel.sh`. I'll remove it and you can adjust the `prepare-for-current-kernel.sh` accordingly.

Comment (Plippo) [reply prepare-for-current-kernel.sh:14]: Thank you for the clarification! No need to remove it, I will adjust it directly.

**Review by Plippo — APPROVED** [review]
Approving after change.

**Comment by mortanius-1** [comment]
Thanks Plippo! All the best.

**Comment by lamualfa** [comment]
Woah. Thanks!

**Pull request merged**